src/scene_map.cpp

```diff
@@ -260,14 +266,12 @@ void Scene_Map::CallBattle() {
 }
 
 void Scene_Map::CallShop() {
-	Game_Temp::shop_calling = false;
```

Comment:
Any reason the Game_Map::GetInterpreter().ResetImmediateCalling is in Game_Interpreter and not in the Call-functions as before?